package.json

```diff
@@ -120,7 +117,7 @@
     "deploy": "storybook-to-ghpages -- --ci",
     "lint": "eslint src *.js",
     "coverage": "yarn run test -- --coverage",
-    "coveralls": "cross-env NODE_ENV=development cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
Since this is meant to run in CI, `cross-env` is superfluous there. Also, with the advent of WSL on newer Windows, running node in Win envs should no longer be a problem (mostly).